platform/ui/src/components/downloadDialog/DownloadDialog.styl

```diff
@@ -0,0 +1,102 @@
+@import './../../design/styles/common/global.styl'
+@import './../../design/styles/common/form.styl'
+@import './../../design/styles/common/button.styl'
+
+.DownloadDialog
+  color: var(--text-secondary-color);
+  filter: drop-shadow(0 0 3px var(--ui-gray-darkest));
+  border: none;
+  border-radius: 8px;
+  width: inherit;
+  padding: 15px;
+  background: transparent;
+  z-index: 1080 !important;
```

Comment:
Why `!important`?

Reply:
In this specific case its because in Modal, the styles were applied in styles attribute directly, it's tricky to override using selectors.

Reply:
Ah, okay. Fine for now. Hopefully modal context and new component will make it easier to clean this up later